### DISCUSSION TITLE
Chia blockchain don't loading plot,farm,nft plot

### DISCUSSION THREAD

**Discussion by Batrakas** [Chia-Network/chia-blockchain#8505] [Support]
Hi,i have made the update,it's synced but my chia blockchain don't load plots,farm and NFT plot.
Thank you.

**Comment by loppefaaret** [comment]
could you check you `config.yaml` file, located under you users folder `~\.chia\mainnet\config\` and check under the
```
pool:
....
  pool_list:
```
section, and tell me if you got anything listed there ?

**Comment by BrandtH22** [comment]
@Batrakas , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
